apps/judicial-system/web/src/routes/Prosecutor/InvestigationRequest/Defendant/Defendant.tsx

```diff
@@ -52,7 +56,9 @@ const Defendant = () => {
                 noNationalId: defendant.noNationalId,
                 citizenship: defendant.citizenship,
               },
-            )
+            ).catch(() => {
```

Comment:
Consider removing this catch and handle the error in the useDefendant hook in a similar way as is done in the useCase hook.